src/app/featureFlags.ts

```diff
@@ -6,6 +6,7 @@ export const flagsForEnvironment = (_: EnvironmentID) => {
     // Banner needs more work to look right and be useful before release - see ticket
     // DON-783
     cookieBannerEnabled: (environment.environmentId === 'development' || environment.environmentId === 'staging'),
+    don819FlagEnabled: (environment.environmentId === 'development' || environment.environmentId === 'staging'),
```

Comment:
Maybe have this only on in dev environment until we want to ask for a non-developer to start reviewing what we've done? I think if it's in staging we might need to make it controllable by the user e.g. via a URL param, so that we can still test the current system with disabled buttons.

```suggestion

    // avoiding use of disabled buttons on donate page where we can and providing more specific error messages instead when we can't continue to the next step. Feature flagged so we can release all steps at once.
    don819FlagEnabled: (environment.environmentId === 'development'),
```